packages/dashboard/src/components/queryEditor/iotSiteWiseQueryEditor/assetModelDataStreamExplorer/assetModelPropertiesExplorer/assetModelPropertiesTable/assetModelPropertiesTable.tsx

```diff
@@ -72,7 +72,7 @@ export function AssetModelPropertiesTable({
     pagination: { pageSize: preferences.pageSize },
     selection: {
       defaultSelectedItems: selectedAssetModelProperties,
-      trackBy: ({ id = '' }) => id,
+      // trackBy: ({ id = '' }) => id,
```

Comment:
nit: remove comments

Reply:
This file needs to be deleted. Old asset model properties table is replaced.